code/modules/vending/clothesmate.dm

```diff
@@ -45,6 +45,12 @@
 		/obj/item/clothing/neck/tie/red = 3,
 		/obj/item/clothing/neck/tie/black = 3,
 		/obj/item/clothing/neck/tie/horrible = 3,
+		/obj/item/clothing/accessory/pride = 3,
+		/obj/item/clothing/accessory/pride/bi = 3,
+		/obj/item/clothing/accessory/pride/trans = 3,
+		/obj/item/clothing/accessory/pride/pan = 3,
+		/obj/item/clothing/accessory/pride/ace = 3,
+		/obj/item/clothing/accessory/pride/enby = 3,
```

Comment:
Could you, for the love of not bloating the clothesmate selection with similar objects, make these into a single reskinnable pin?